src/components/__tests__/Panel.js

```diff
@@ -30,6 +30,21 @@ describe('default render', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
+  it('passes data without refinement', () => {
```

Comment:
We can add another test where there are refinements and `noRefinement` is `false`.